templates/_github/workflows/dependabot-auto-merge.raw.yml

```diff
@@ -4,19 +4,14 @@
 name: Auto-Merge Dependabot PRs
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
This is correct, but dangerous. We should add a comment to warn users not to add anything here which needs write access.